stl/inc/mutex

```diff
@@ -522,7 +521,21 @@ auto _Invoke_stored(tuple<_Types...>&& _Tuple)
     return _Invoke_stored_explicit(_STD move(_Tuple), index_sequence_for<_Types...>());
 }
 
+// STRUCT once_flag
+struct once_flag { // opaque data structure for call_once()
+    constexpr once_flag()       = default;
+    once_flag(const once_flag&) = delete;
+    once_flag& operator=(const once_flag&) = delete;
+
+    void* _Opaque = nullptr;
+};
+
 // FUNCTION TEMPLATE call_once
+using _Execute_once_fp_t = int(__stdcall*)(void*, void*, void**);
+
+_CRTIMP2_PURE int __CLRCALL_PURE_OR_CDECL _Execute_once(

```

Comment:
`_NODISCARD`?

Reply:
I don't think it's worth figuring out the mess of interaction with the dllexport stuff. It has exactly one callsite, so that will never find a bug.